sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -271,7 +271,7 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      *
      * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguageBatch#Iterable-TextAnalyticsRequestOptions}
      *
-     * @param textInputs The list of {@link DetectLanguageInput inputs/documents} to be analyzed.
+     * @param documents The list of {@link DetectLanguageInput inputs/documents} to be analyzed.
```

Comment:
I think it would be better just to refer to it as documents instead of inputs/documents